features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunBorderRouter.h

```diff
@@ -17,6 +17,37 @@
 #ifndef WISUNBORDERROUTER_H
 #define WISUNBORDERROUTER_H
 
+/**
+ * \brief Struct br_information Border router dynamic information.
+ */
+typedef struct br_information {
+    /** Address prefix given to devices in network  set to 0 if not available*/
+    uint8_t ipv6_prefix[8];
+    /** IID of Border router */
+    uint8_t ipv6_iid[8];
+    /** Border router dodag id */
+    uint8_t rpl_dodag_id[16];
+    /** Border router instance identifier defined in RPL */
+    uint8_t instance_id;
+    /** RPL version number */
+    uint8_t version;
+    /** Timestamp of the the device. Can be used as version number*/
+    uint64_t host_timestamp;
+    /** Amount of devices in the network. */
+    uint16_t device_count;
+} br_information_t;
```

Comment:
To avoid possible name clashes in the future, should there be a prefix in the structure name like `ws_br_information_t` , `ws_br_route_info_t` or similar